### PR TITLE
Add variable definitions for chapter 8

### PR DESCRIPTION
**Pull request by danielhuppmann**
Adding the variables for chapter 7 based on the data received so far, with some minor spelling harmonizations. Please review the descriptions

### CODE REVIEW

File: definitions/variable/tag_land-removal.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- Land Removal:
+    - Land:
+        description: land removal
```
Comment (gidden) [inline definitions/variable/tag_land-removal.yaml:3]: all carbon removal on land, including both conventional CDR and removals due to indirect carbon fluxes aligned with national greenhouse gas inventories

File: definitions/variable/tag_land-removal.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    - Land|Direct:
```
Comment (gidden) [inline definitions/variable/tag_land-removal.yaml:4]: `Land|Conventional CDR`

File: definitions/variable/tag_land-removal.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+        description: direct land removal
```
Comment (gidden) [inline definitions/variable/tag_land-removal.yaml:5]: conventional carbon removal on land excluding shifting cultivation

File: definitions/variable/tag_land-removal.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+    - Land|Indirect:
+        description: indirect land removal
```
Comment (gidden) [inline definitions/variable/tag_land-removal.yaml:7]: carbon removal on land due to indirect carbon fluxes accounted for in NGHGIs but not by scientific models

File: definitions/variable/variables.yaml [hunk 5]
```diff
@@ -1,3 +1,35 @@
...
     unit: Mt CO2/yr
+
+- Gross Emissions|CO2 [{Quarter-Percentile}]:
+    description: Gross emissions of carbon dioxide (CO2), not accounting for negative
+      emissions from bioenergy with CCS (BECCS) or agriculture, forestry and other land
+      use (AFOLU), given as the {Quarter-Percentile} of a scenario ensemble
+    unit: Mt CO2/yr
+    note: For statistical indicators, the scenario-name column in the IAMC format should
+      refer to the scenario ensemble or selection criteria of scenarios
+- Carbon Removal [{Quarter-Percentile}]:
```
Comment (gidden) [inline definitions/variable/variables.yaml:12]: I do not see this defined anywhere in this PR, but want to confirm it is the sum of novel CDR and conventional CDR (conventional defined in land-removal)

**Review by gidden — CHANGES_REQUESTED** [review]
Thanks so much @danielhuppmann - I added some comments for update.

File: definitions/variable/tag_land-removal.yaml (hunk 4 above)
Comment (danielhuppmann) [reply definitions/variable/tag_land-removal.yaml]: Done

File: definitions/variable/tag_land-removal.yaml (hunk 1 above)
Comment (danielhuppmann) [reply definitions/variable/tag_land-removal.yaml]: Done

File: definitions/variable/variables.yaml (hunk 5 above)
Comment (danielhuppmann) [reply definitions/variable/variables.yaml:12]: I don't see a variable like `Carbon Removal|Land` or subcategories of it in the provided data.

Comment (gidden) [reply definitions/variable/variables.yaml:12]: @gaurav-ganti - could you help check on this please?

**Comment by danielhuppmann** [comment]
Chapter 7 was changed to Chapter 8

**Comment by gidden** [comment]
Thanks again so much @danielhuppmann. In reviewing the data file you sent and what is in our figure data sheet, I have the following requests to variable names:

1. AR6 Reanalysis|OSCARv3.2|Carbon Removal|Land|Conventional CDR  -> Carbon Removal|Conventional
2. Gross Emissions|CO2 -> Gross Emission Reductions from 2020|CO2

**Comment by danielhuppmann** [comment]
Thanks for the comments, @gidden. In the interest of readability, can I suggest an alternative?

> Gross Emissions|CO2 -> ~Gross Emission Reductions from 2020|CO2~ Gross Emissions|CO2 [Reductions relative to 2020]

**Comment by danielhuppmann** [comment]
And to avoid any confusion, @gidden - should the variables
- *AR6 Reanalysis|OSCARv3.2|Carbon Removal|Land*
- *AR6 Reanalysis|OSCARv3.2|Carbon Removal|Land|Indirect*

also be shortened similar to your suggestion for *AR6 Reanalysis|OSCARv3.2|Carbon Removal|Land|Conventional CDR*?

**Comment by gidden** [comment]
No I would prefer to keep those variables as is. They are relevant for Figure 7.4, but not for any indicator (they mix flux types which should not be misunderstood by users/policy makers).

File: definitions/variable/variables.yaml (hunk 5 above)
Comment (gidden) [reply definitions/variable/variables.yaml:56]: Ok, I think this is clear now

**Pull request merged**